implementations/wrapscan-registry/polywrap.graphql

```diff
@@ -0,0 +1,7 @@
+#import * into UriResolver from "wrap://ens/wraps.eth:uri-resolver-ext@1.1.0"
+
+type Module implements UriResolver_Module {}
+
+type Env {
+  resolveUrl: String
```

Comment:
I'd actually name this `providerUrl` or `gatewayUrl`.
Ofc, this means the `/r` would be hardcoded in the resolver itself.
This would allow us to dynamically update it later (as opposed to having to re-release clients if the route changes)